Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
+      event_sourcery (>= 0.14.0)
       pg
-      sequel (~> 4.38)
-    highline (1.7.8)
-    method_source (0.8.2)
-    mustermann (1.0.0)
+      sequel (>= 4.38)
+    highline (2.0.0)
```

Comment:
### [highline](https://github.com/JEG2/highline)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 1.7.8 → 2.0.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **596** commits. See the full changes on [the compare page](https://github.com/JEG2/highline/compare/v1.7.8...v2.0.0).

These are the first 10 commits:
- (a947396) [Add ExplainableError class and take advantage of inheritance…](https://github.com/JEG2/highline/commit/a9473962a4f2c07082e51c08c1be9421c0005608)
- (86ebb0d) [Makes QuestionError a ExplainableError](https://github.com/JEG2/highline/commit/86ebb0d936e9da876c8f4cccdb5591bea6404524)
- (c76976a) [Extract Menu#get_item_by_number and #get_item_by_letter](https://github.com/JEG2/highline/commit/c76976aa2620b01992c3f34becd32953f9f96caa)
- (a1b9acd) [Add some comments to Menu methods](https://github.com/JEG2/highline/commit/a1b9acd904c8dfcd6956d7f7216ed3a0f6ad0538)
- (c433c37) [Extract HighLine#shell_style_lambda](https://github.com/JEG2/highline/commit/c433c37a41675cf151433574ec4e404ac04d6ab2)
- (d5ad359) [Remove unecessary ins vars assignment and make use of access…](https://github.com/JEG2/highline/commit/d5ad3597b9f550e26953bdf6f1cc3e86048fee9c)
- (7ba5aa2) [Merge pull request #178 from abinoam/feat_improve_on_code_cl…](https://github.com/JEG2/highline/commit/7ba5aa219d9058a8d46d24f39efcc36bcf6a87ee)
- (e85d4c0) [Improve HighLine::Menu#choice documentation](https://github.com/JEG2/highline/commit/e85d4c082fa49851062fe163eaffe56dd08fe217)
- (a687cc2) [Improve HighLine::Menu#choices documentation](https://github.com/JEG2/highline/commit/a687cc2af40b09715f69f5fc94728a83b4ebd394)
- (6c399e0) [Improve HighLine::Menu#hidden documentation](https://github.com/JEG2/highline/commit/6c399e025e14bc8cbb8ac31b056997b08b31bca6)


</details>